entity-framework/core/testing/in-memory.md

```diff
@@ -1,24 +0,0 @@
----
```

Comment:
Is it okay to just delete this? Won't that result in 404s for existing links?

Reply:
I added a redirection here to the testing index here: https://github.com/dotnet/EntityFramework.Docs/pull/3539/files#diff-3fa6f2a8672a06f5c881c8621af9e49f935e65eb99e06ec73d42afe0b1323d7aR459, is that enough?